go/internal/ocxcli/families.go

```
@@ -0,0 +1,1605 @@
+
+func takeFlag(args *[]string, flag string) bool {
+	for i, value := range *args {
+		if value == flag {
+			*args = append((*args)[:i], (*args)[i+1:]...)
+			return true
+		}
+	}
+	return false
+}
+
+var blockedConfigSegment = map[string]bool{"__proto__": true, "prototype": true, "constructor": true}
```

golangci-lint (2.13.2)

[error] var blockedConfigSegment is unused

(unused)